README.md

```diff
@@ -15,7 +15,28 @@ npm run build-cdn
 
 ## Usage
 
-todo, add examples
+Simply include the Highlight.js library in your webpage or Node app, then load this module.
+
+### Static website or simple usage
+
+Simply load the module after loading Highlight.js. This module is just a CDN build of the language, so it will register itself as the Javascript is loaded.
+
+```html
+<link rel="stylesheet" href="/path/to/styles/default.css">
+<script src="/path/to/highlight.min.js"></script>
+<script src="/path/to/highlightjs-sap-abap/dist/abap.min.js"></script>
+<script type="text/javascript">
+  hljs.initHighlightingOnLoad();
```

Comment:
Please use use v11 API. `highlightAll()`, etc.

Reply:
done in https://github.com/highlightjs/highlightjs-sap-abap/pull/8